WORKSPACE

```diff
@@ -0,0 +1,64 @@
+workspace(name = "com_lucidchart_rules_hocon")
```

Comment:
I would recommend running `buildifier` to keep `BUILD.bazel` and `WORKSPACE` file formatting consistent.
https://github.com/bazelbuild/buildtools/tree/master/buildifier

Reply:
done